src/rf/apps/workers/process.py

```diff
@@ -58,48 +56,53 @@ def start_polling(self):
                         delete_message = self.check_timeout(record['data'])
                     elif job_type in JOB_VALIDATE:
                         if record['eventSource'] == 'aws:s3':
-                            delete_message = self.validate_image(record)
+                            delete_message = self._validate_image(record)
 
                 if delete_message:
-                    self.q.delete_message(message)
+                    self.queue.remove_message(s3_message)
 
-    def validate_image(self, data):
+    def _validate_image(self, data):
         """
         Use Gdal to verify an image is properly formatted and can be further
         processed.
         data -- attribute data from SQS.
         """
-        # Pass image to Gdal to verify.
-        # If it succeeds continue else return False.
+        def mark_image_valid(s3_uuid):
+            image = LayerImage.objects.get(s3_uuid=s3_uuid)
+            image.status = enums.STATUS_VALID
```

Comment:
There's at least one other place in this file where you need to update satus_updated_at.